Ch9/Ejemplo/index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Form</title>
+    <meta 
+        charset="UTF-8" />
+    <meta 
+        name="viewport" 
+        content="width=device-width, initial-scale=1" />
+  </head>
+  <body>
+    <header>
+      <h1>
+        Ejemplo de Formulario
+    </h1>
+    </header>
+    <form>
+      <h3>
+        Placeholder y Autofocus
+      </h3>
+      <input 
+        autofocus
+        name="a" 
+        placeholder="Placeholder y Autofocus" />
+    </form>
+    <form>
+        <h3>
+            Funciones clásicas
+        </h3>
+        <input 
+            type="checkbox">
+        <input 
+            type="radio">
+        <input 
+            type="password">
+        <select>
+            <option 
+                value="1">
+                    a
+            </option>
+            <option 
+                value="2">
+                    b
+            </option>
+        </select>
+        <input 
+            type="file">
+        <input 
+            type="submit">
+        <input 
+            type="text">
+    </form>
+    <form>
+        <h3>
+            Email
+        </h3>
+        <input 
+            type="email">
+    </form>
+    <form>
+        <h3>
+            Web
+        </h3>
+        <input 
+            type="url">
+    </form>
+    <form>
+        <h3>
```

Comment:
además de etiquetas de headings también como ejemplo puedes ocupar etiquetas label que tienen un atributo especial para los forms que es el `for`

por ejemplo:

```html
<label for=spinbox1>
  Spinbox
</label>
<input 
  id=spinbox1
  type=checkbox>
```